src/Apps/Consign/consignFromCollectorProfileMyCollectionRoutes.tsx

```diff
@@ -5,26 +5,6 @@ import { AppRouteConfig } from "System/Router/Route"
 import { getENV } from "Utils/getENV"
 import { getRedirect } from "./Routes/SubmissionFlow/Utils/redirects"
 
-const MarketingLandingApp = loadable(
```

Comment:
I wonder if all the remaining routes within this file should be moved to `src/Apps/CollectorProfile/collectorProfileRoutes.tsx` because they all start with `/collector-profile/`... 🤔

Reply:
Oh wow, we have a lot of routes 😅 I think yes, we can move them to collectorProfileRoutes. I'll do it and test